neuland/digitizing/R3BDigitizingPaddle.cxx

```diff
@@ -40,6 +40,26 @@ namespace R3B::Digitizing
         }
     }
 
+        //Paula: new SetChannel to have different Parameters for every paddleID
+    void Paddle::SetChannel(std::unique_ptr<Channel> channel, int Module_ID)
+    {
+        channel->SetPaddle(this);
+        channel->SetPar(Module_ID);
+        channel->AttachToPaddle(this);
+        if (channel->GetSide() == ChannelSide::left)
+        {
+            fLeftChannel = std::move(channel);
+        }
+        else if (channel->GetSide() == ChannelSide::right)
+        {
+            fRightChannel = std::move(channel);
+        }
+        else
+        {
+            LOG(error) << "Channel side is invalid!";
+        }
+    }
+    
```

Comment:
Same as above. The old `SetChannel` already gives the channel the information about module_id.